### ISSUE TITLE
Update / Roadmap for this?

### ISSUE THREAD

**Issue by JohanSJA** [ASoftCo/lita-facebook#1]
The repo is just barebone for now. Any plan to share?

**Comment by sergey-alekseev** [comment]
@JohanSJA yeah, going to write an adapter soon. Will update you here. Do you want to participate?

**Comment by estiens** [comment]
Any word on this?

**Comment by sergey-alekseev** [comment]
@estiens unfortunately not. Do you want to take care of it?
